model/task_lifecycle_test.go

```diff
@@ -37,6 +37,8 @@ var (
 func TestSetActiveState(t *testing.T) {
 	Convey("With one task with no dependencies", t, func() {
 		require.NoError(t, db.ClearCollections(task.Collection, build.Collection, task.OldCollection, VersionCollection))
+		require.NoError(t, db.CreateCollections(task.Collection, task.OldCollection))
```

Comment:
no need to explicitly create task collections if we insert a task 
but it might be good to clear old collection

Reply:
Fixed!